service/dynamodb/expression/condition_test.go

```diff
@@ -16,12 +16,14 @@ type testCompare interface {
 //Equal
 func TestCompare(t *testing.T) {
 	cases := []struct {
+		name     string
 		lhs      testCompare
 		rhs      OperandBuilder
 		mode     ConditionMode
 		expected Expression
 	}{
 		{
+			name: "nested path with path",
 			lhs:  NewPath("foo.yay.cool.rad"),
 			rhs:  NewPath("bar"),
```

Comment:
these probably can be simplified to use `input` like the other test methods instead of `lhs` and `rhs` This also removes the `EqualsCond.